spec/core_functions/color/invert.hrx

```diff
@@ -314,3 +407,65 @@ Error: argument `$weight` of `invert($color, $weight: 100%)` must be between -0
 >> a {b: invert(red, 100.001)}
 
    ------^
+
+<===>
+================================================================================
+<===> error/non_legacy/null_space/options.yml
+:ignore_for:
+  - libsass
+
+<===> error/non_legacy/null_space/input.scss
+a {b: invert(lch(54.3 107 40.9))}
+
+<===> error/non_legacy/null_space/error
+Error: $color: To use color.invert() with non-legacy color lch(54.3% 107 40.9deg), you must provide a $space.
+  ,
+1 | a {b: invert(lch(54.3 107 40.9))}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/non_one_hundred_weight/options.yml
+:ignore_for:
+  - libsass
+
+<===> error/non_one_hundred_weight/input.scss
+a {b: invert(lch(54.3 107 40.9), 92%, s)}
+
+<===> error/non_one_hundred_weight/error
+Error: $space: Unknown color space "s".
+  ,
+1 | a {b: invert(lch(54.3 107 40.9), 92%, s)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/unknown_space/options.yml
+:ignore_for:
+  - libsass
+
+<===> error/unknown_space/input.scss
+a {b: invert(lch(54.3 107 40.9), $space: s)}
+
+<===> error/unknown_space/error
+Error: $space: Unknown color space "s".
+  ,
+1 | a {b: invert(lch(54.3 107 40.9), $space: s)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/invalid_method/options.yml
+:ignore_for:
+  - libsass
+  
+<===> error/invalid_method/input.scss
+a {b: invert(turquoise, $space: display-p3)}
```

Comment:
`display-P3` is not listed as one of the valid [color interpolation methods](https://www.w3.org/TR/css-color-4/#interpolation-space). Should this error?